Editorial fixes: Address Barry's and Robert's comments (#96)
ietf-tapswg/draft-ietf-taps-transport-security: draft-ietf-taps-transport-security.md

document supplements that work by surveying commonly used and notable network security protocols,
and identifying the interfaces between these protocols and both transport protocols and
applications.  It examines Transport Layer Security (TLS), Datagram Transport Layer Security (DTLS),
IETF QUIC, Google QUIC (gQUIC), tcpcrypt, Internet Protocol Security (IPsec), Secure Real-time Transport Protocol (SRTP) with DTLS,
WireGuard, CurveCP, and MinimaLT. For each protocol, this document provides a brief description.
Then, it describes the interfaces between these protocols and transports in {{transport-interface}}
and the interfaces between these protocols and applications in {{application-interface}}.

> A Transport Services system exposes an interface to different Transport Services to applications. Further, a Transport Services system maps the Transport Service that an application has requested to a specific protocol stack, including transport protocols and security protocols.

Reviewer: ```suggestion
A Transport Services system exposes an interface for applications to access various transport protocol features, including transports that offer security.
```

Reviewer 2: How about the following?

"A Transport Services system exposes an application interface for accessing various (secure) transport protocol features."

Reviewer 3: Thanks for the suggestions! I like both and I'm going with the latter.

> The security protocols included in survey represent a superset of functionality and features a Transport Services system may

Reviewer: ```suggestion
The security protocols included in this survey represent a superset of functionality and features a Transport Services system may
```

IPsec. A major goal of this protocol is to provide a VPN that is simple to
configure and works over a variety of transports. OpenVPN encapsulates either
IP packets or Ethernet frames within a secure tunnel and can run over either UDP or TCP.

> For key establishment, OpenVPN can use TLS as a handshake protocol or it can use pre-shared keys.

Reviewer: ```suggestion
For key establishment, OpenVPN can either use TLS as a handshake protocol or use pre-shared keys.
```
